src/ra.erl

```diff
@@ -928,19 +928,38 @@ pipeline_command(ServerId, Command) ->
 local_query(ServerId, QueryFun) ->
     local_query(ServerId, QueryFun, ?DEFAULT_TIMEOUT).
 
-%% @doc Same as `local_query/2' but accepts a custom timeout.
+%% @doc Same as `local_query/2' but accepts a custom timeout or a map of
+%% options.
+%%
+%% The supported options are:
+%% <ul>
+%% <li>`wait_for_index': the query will be evaluated only once the specific
```

Comment:
I see. I will rename it back to `limit`.

Should the argument be of the form `{limit, {Index, Term}}` then? To give a bit more context about what the limit is.

Reply:
I think it is ok to leave it as {Index, Term} for now as a theoretical shorthand for a future `{applied, {Index, Term}}` type of definition. Or we could call the argument `condition` and always use a longer from.

I was thinking that perhaps we'd like a condition around the effective machine version. It would be good to be able to detect the scenario where a member has a lower machine version than the effective one and respond quickly to that

Reply:
I renamed the option without changing the value format for now.